tasks/apt.yml

```diff
@@ -15,7 +15,6 @@
       - libffi-dev
       - python-dev
       - redis-server
-    state: latest
```

Comment:
why is this removed? 
Is this considered a bad practice? CC @hispanico

Reply:
i've get linter error for state latest and i do deleted all latest state

Reply:
30 E403:  Package installs should not use latest